hortum_mobile/pubspec.yaml

```diff
@@ -78,7 +80,6 @@ flutter:
         - asset: assets/fonts/Roboto-Light.ttf
         - asset: assets/fonts/Roboto-Bold.ttf
         - asset: assets/fonts/Roboto-Medium.ttf  
```

Comment:
Foram adicionados mais arquivos de fonte na pasta "assets" e nem todos serão usados ou foram chamados aqui, então pode remover aqueles que não serão usados, como as fontes em itálico.

Reply:
Retirado fonts extras da pasta assets